### DISCUSSION TITLE
skyline.github.com results in flat skyline

### DISCUSSION THREAD

**Discussion by martijnvanonz** [community/community#43476] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

The skyline does not work anymore. The base renders but the skyline is flat for all users:
https://skyline.github.com/martijnvanonz/2022
https://skyline.github.com/frenck/2022

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @martijnvanonz ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by radar-blip** [comment]
Maybe they reset it for the new year.

**Reply by martijnvanonz** [reply]
It was working last week. I wanted to print the 2022 when it was finished. 2021 is also flat.

**Reply by radar-blip** [reply]
Not sure. I just assumed it had something to do with new year. Maybe its temporarily broke.
